code/src/java/pcgen/cdom/content/factset/FactSetDefinitionEnforcer.java

```diff
@@ -79,7 +72,7 @@ public boolean process(LoadContext context, T obj)
 		 * Note, even if the Indirects in list are empty this should pass,
 		 * because they TRIED, right?
 		 */
-		if ((list != null) && !list.isEmpty())
+		if (list != null && !list.isEmpty())
```

Comment:
I think the removal of parenthesis here removes clarity for a reader.  I recognize they are not technically required, but I also don't think something should be removed just because it can be removed.